backend/mark_marker/src/main/scala/mark_marker/trainer/DatabaseConnector.scala

```diff
@@ -0,0 +1,96 @@
+package mark_marker.trainer
+
+import java.io._
+import java.sql.DriverManager
+import java.sql.Connection
+
+import edu.stanford.nlp.classify.{Classifier, ColumnDataClassifier}
+
+import scala.io.Source
+
+class DatabaseConnector {
+  private val driver = "com.mysql.cj.jdbc.Driver"
+  private val url = "jdbc:mysql://tc.jstudios.ovh:3306/Automarker"
+  private val username = "python"
+  private val psw = findPsw("../dbpass.txt")
+  private var connection: Connection = _
+
+  def findPsw(path: String) = {
+    Source.fromFile(new File(path)).mkString
+  }
+
+  def connect() = {
+    Class.forName(driver)
+    DriverManager.getConnection(url, username, psw)
+    connection = DriverManager.getConnection(url, username, psw)
+  }
+
+  def close() = connection.close()
+
+  def getTrainingData(exercise: String): String = {
+    val stat = connection.prepareStatement(s"SELECT training_set FROM MarkMarkerTraining WHERE exercise_name = ?")
+    stat.setString(1, exercise)
+    val resultSet = stat.executeQuery
+    val sb = new StringBuilder
+    while (resultSet.next)
+      sb.append(resultSet.getString("training_set"))
+    resultSet.close()
+    sb.toString()
+  }
+
+  def getSerialisedClassifier(exercise: String): Classifier[String, String] = {
```

Comment:
I wasn't sure whether to store the big training set or just line per line.
We will probably need a separate table to hold the serialized blob however.